src/libcore/slice/mod.rs

```diff
@@ -5419,6 +5453,153 @@ unsafe impl<'a, T> TrustedRandomAccess for ChunksExactMut<'a, T> {
     }
 }
 
+/// An iterator over a slice in (non-overlapping) chunks (`N` elements at a
+/// time), starting at the beginning of the slice.
+///
+/// When the slice len is not evenly divided by the chunk size, the last
+/// up to `chunk_size-1` elements will be omitted but can be retrieved from
+/// the [`remainder`] function from the iterator.
+///
+/// This struct is created by the [`array_chunks`] method on [slices].
+///
+/// [`array_chunks`]: ../../std/primitive.slice.html#method.array_chunks
+/// [`remainder`]: ../../std/slice/struct.ArrayChunks.html#method.remainder
+/// [slices]: ../../std/primitive.slice.html
```

Comment:
These links will be broken if viewed from the `core` docs. Unfortunately `slice` can't be fixed due to #63351.
```suggestion
/// [`remainder`]: ArrayChunks::remainder
/// [slices]: ../../std/primitive.slice.html
```